app/helpers/checkout_helper.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 module CheckoutHelper
+  def shipping_and_billing_match?(order)
+    order.ship_address == order.bill_address
+  end
+
```

Comment:
This doesn't actually do the same thing as the extracted method; if the two addresses have all the same attributes but different ID's, this returns false. In that scenario I think we should consider them matching addresses.

Reply:
> if the two addresses have all the same attributes but different ID's, this returns false

The `Spree::Address` class has it's own implementation of the `==` method, so it should return `true` in that scenario:

https://github.com/openfoodfoundation/openfoodnetwork/blob/3e0509d23241dd0ae71f02f1f221a78ef03a0b3b/app/models/spree/address.rb#L62-L71

Reply:
Ahh nice, good catch. That makes this way cleaner 👍